torch/testing/_internal/common_methods_invocations.py

```diff
@@ -807,9 +807,9 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
     UnaryUfuncInfo('atanh',
                    ref=np.arctanh,
                    domain=(-1, 1),
-                   dtypes=all_types_and(torch.bool),
-                   dtypesIfCPU=all_types_and(torch.bool),
-                   dtypesIfCUDA=all_types_and(torch.bool, torch.half, torch.bfloat16),
+                   dtypes=all_types_and_complex_and(torch.bool),
```

Comment:
Like other trig operators these will probably need to skip the reference numerics test for complex on CPU. In the future we should revise that test to have an easier version, like test_small_reference_numerics, so we only have to skip some values in testing.